src/server/pfs/server/driver.go

```diff
@@ -1625,7 +1625,7 @@ func (d *driver) clearCommit(ctx context.Context, commit *pfs.Commit) error {
 	if err != nil {
 		return err
 	}
-	if commitInfo.Finished != nil {
+	if commitInfo.Finishing != nil {
 		return errors.Errorf("cannot clear finished commit")
```

Comment:
Does this error need to update to reflect the changed behavior?

Reply:
Not sure, the clients are exposed to this process through finish commit. The finishing without finished state is just something that causes waiting under the hood.